server/controllers/recommendation/redirectController.test.ts

```diff
@@ -211,8 +211,8 @@ describe('get', () => {
     expect(next).toHaveBeenCalled()
   })
 
-  it('redirect to task-list-no-recall if SPO_CONSIDERING_RECALL', async () => {
-    ;(getStatuses as jest.Mock).mockResolvedValue([{ name: 'SPO_CONSIDERING_RECALL', active: true }])
```

Comment:
Is it this that has prompted you to use the STATUSES enum? i.e. perhaps this used to be called SPO_CONSIDERING_RECALL but is now SPO_CONSIDER_RECALL? Or perhaps it's not that it's changed, it's just a bug in the tests.

If SPO_CONSIDERING_RECALL no longer exists, there are still a couple of references to it in the project that probably ought to be removed.